src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -304,10 +318,10 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 					resource: undefined,
 					options: {
 						pinned: true,
-						index: this.group.count, // always at the end
+						index: this.groupViewer.count, // always at the end
 						override: DEFAULT_EDITOR_ASSOCIATION.id
 					}
-				}, this.group.id);
+				}, this.tabsModel.id);
```

Comment:
I know we can use it, but I think in all cases of `this.tabsModel.id` we should use `this.groupViewer.id` simply because we always want to refer to the ID of the group and that would make it more explict.